auth/localkeys.go

```diff
@@ -33,8 +33,8 @@ const (
 var (
 	delegateKeys HostKeys
 	masterKeys   MasterKeys
-
-	dKeyCond = &sync.Cond{L: &sync.Mutex{}}
+	mKeyLock     sync.Mutex
```

Comment:
This might not make a huge difference, but I recommend using a sync.RWMutex here. The writes are very infrequent, so you can usually RLock rather than Lock.